contracts/setup_blending.sh

```diff
@@ -123,8 +144,11 @@ register_tokens() {
 }
 
 set_up_blending() {
-  log_progress "Launching local chain..."
-  run_chain || error "Failed to launch chain"
+
+  if [ $RUN_CHAIN = true ]; then
+    log_progress "Launching local chain..."
+    run_chain || error "Failed to launch chain"
```

Comment:
`run_chain` calls `./scripts/run_nodes.sh` -- why not run the smarknet?

Reply:
I don't know, some prefer binary